src/main/java/CurlLoggingFilter.java

```diff
@@ -0,0 +1,98 @@
+import jakarta.servlet.Filter;
```

Comment:
The import should be 'javax.servlet.Filter' instead of 'jakarta.servlet.Filter'.